src/longRunningCalls/longrunning.ts

```diff
@@ -299,7 +310,11 @@ export class Operation extends EventEmitter {
           if (
             rawResponse!.metadata &&
             (!previousMetadataBytes ||
-              !rawResponse!.metadata!.value.equals(previousMetadataBytes))
+              (rawResponse &&
+                !arrayEquals(
```

Comment:
what happens if two identical sets of bytes are served? what is this protecting against exactly.

Reply:
We want to emit an event only if the metadata contents has changed. This change does not change behavior, it just makes the existing code work in browser context (where there might be no `Buffer` available).